packages/react/src/components/TearSheet/TearSheetWrapper.jsx

```diff
@@ -17,27 +16,22 @@ const propTypes = {
   children: PropTypes.node,
 };
 
-const tearSheetConstants = {
-  DISTANCE_FROM_TOP: 88,
-  WIDTH_DECREASE: 24,
-  BOTTOM_POSITION_WHEN_HIDDEN: 12,
-  DISTANCE_FROM_EACH_SIDE: 64,
-};
-
 const defaultProps = {
   isOpen: false,
   className: undefined,
   onCloseAllTearSheets: undefined,
   children: undefined,
 };
 
+const baseClassName = `${iotPrefix}--tear-sheet-wrapper`;
+
 const TearSheetWrapper = ({ isOpen, className, onCloseAllTearSheets, children }) => {
-  const windowSize = useWindowSize();
+  // const windowSize = useWindowSize();
```

Comment:
Should we remove or uncomment these lines?

Reply:
removed comments